test/nbrowser/Localization.ts

```diff
@@ -51,8 +51,8 @@ describe("Localization", function() {
     assert.deepEqual(gristConfig.namespaces.sort(), [...namespaces].sort());
   });
 
-  // Now make a Polish language file, and test that it is used.
-  describe("with Polish language file", function() {
+  // Now make a uz-UZ language file, and test that it is used.
+  describe("with uz-UZ language file", function() {
```

Comment:
I see that uz has a lot fewer translations than pl on weblate, but there are still some, so this feels a bit like moving a bubble around. Is it possible to use an entirely spurious code or will something complain?